roles/setup_assisted_installer/tasks/main.yml

```
@@ -40,14 +40,23 @@
     - name: Open ports zone internal and public, for firewalld
       firewalld:
         port: "{{ item.1 }}/tcp"
-        permanent: yes
-        immediate: yes
+        permanent: true
+        immediate: true
         state: enabled
         zone: "{{ item.0 }}"
       loop: "{{ ['internal', 'public'] | product(['8000', port, assisted_service_gui_port, assisted_service_image_service_port]) | list }}"
 
-    - name: Create directories for assisted-installer
+    - name: Delete previous assisted-installer directories
       file:
+        path: "{{ item }}"
+        state: absent
+        force: true
+      with_items:
+        - "{{ assisted_installer_dir }}"
```

```suggestion
        - "{{ assisted_installer_data_dir }}"
        - "{{ assisted_installer_dir }}"
```

it could be overridden to not be in assisted_installer_dir